roles/argocd/templates/argocd-configmap.yaml.j2

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-cm
+  namespace: argocd
```

Comment:
Shouldn't this be `{{ argocd_namespace }}`?

Reply:
Done